app/controllers/dinosaurs_controller.rb

```diff
@@ -6,4 +6,30 @@ def index
   def show
     @dinosaur = Dinosaur.find(params[:id])
   end
+
+  def new
+    @dinosaur = Dinosaur.new
+  end
+
+  def create
+    @dinosaur = Dinosaur.new(dinosaur_params)
+
+    user = FactoryGirl.create(:user)
+    @dinosaur.user = user
```

Comment:
As tempting as it may be, don't use Factory Girl here. If dinosaur creation is dependent upon associating a user with it and that user being logged in, figure out how to do that with devise before proceeding; it's a dependency of the feature. The tests should also reflect that dependency (i.e. user cannot create dinosaur unless logged in, etc).